src/caffe/proto/caffe.proto

```diff
@@ -499,6 +499,8 @@ message HDF5DataParameter {
   optional string source = 1;
   // Specify the batch size.
   optional uint32 batch_size = 2;
+  // Specify shuffling the order of training data
```

Comment:
Please clarify this comment to explain that the HDF5 files themselves are shuffled but the order within any given file is fixed.